test/unit/publishProjectWebViewController.test.ts

```diff
@@ -349,56 +366,55 @@ suite("PublishProjectWebViewController Tests", () => {
             Var2: "Value2",
         });
 
-        // Verify deployment options were loaded from DacFx
+        // Verify deployment options were loaded from DacFx matching XML properties
         expect(mockDacFxService.getOptionsFromProfile.calledOnce).to.be.true;
+        expect(newState.deploymentOptions.excludeObjectTypes.value).to.deep.equal([
+            "Users",
+            "Logins",
+        ]);
+        expect(newState.deploymentOptions.booleanOptionsDictionary.allowIncompatiblePlatform?.value)
+            .to.be.true;
+        expect(newState.deploymentOptions.booleanOptionsDictionary.ignoreComments?.value).to.be
+            .true;
```

Comment:
Can you add failure messages to these like you did for "updateDeploymentOptions reducer should be registered"?